### ISSUE TITLE
Element for Text

### ISSUE THREAD

**Issue by alensiljak** [gaphor/gaphor#2735]
### Is your feature request related to a problem?
I'm trying to add a title text to a diagram and this element does not exist.

### Describe the solution you'd like
A new element in the General section, for Text. It should allow adding Text to the diagram. Additionally, it would allow formatted text and adding borders to the container box.

### Describe alternatives you've considered
Using Note/comment element. Looks weird, though. Using other elements adds the element to the model.

### Additional context
I'm used to this from Enterprise Architect and other UML tools. Usually I add the start diagram with relevant titles and links.

**Comment by amolenaar** [comment]
@alensiljak Thank you for the feature request. I think adding "just text" would be a nice addition.

Do you want to help implement it?

**Comment by alensiljak** [comment]
I'd sure love to look into it. I've just discovered this amazing project and would love to help. Need to squeeze it amongst the multiple other things going on but this is a keeper.
I've worked with Enterprise Architect in the past and have been looking for an alternative for OS projects for years. This one ticks all the boxes so far.

**Comment by dmkeijzer** [comment]
I was looking for exactly this feature as I think it would be a great addition and a simple textbox would go a long way to enhancing the usability. @alensiljak  Have you managed to nearly finish it by any chance?

**Comment by alensiljak** [comment]
No, unfortunately, I did not really understand the domain model and later I no longer had time to invest into learning another project and technology.

**Comment by dmkeijzer** [comment]
I understand, for a temporary solution you remove the outlining by using the dash-style in the style sheets. I'm currently using this for anyone that is interested. 

```json
comment {
 /* line-style: sloppy 0.3; */
 opacity: 1;
 font-size: 10;
 line-width: 2;
 dash-style: 1 2000;
 }
```
